### ISSUE TITLE
Missing form label on user profile page

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#370]
On https://dmptool.org/users/edit

<input placeholder="Please enter the institution name" class="form-control toggleable-field user-entered-org-a38ae7bb9e0d" maxlength="80" size="80" type="text" name="org_autocomplete[user_entered_name]" id="org_autocomplete_user_entered_name" style="display: none;">

Applicable WCAG Standard(s)
1.1.1 Non-text Content (Level A)
1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)
3.3.2 Labels or Instructions (Level A)
